OOModel/src/declarations/MetaDefinition.h

```diff
@@ -41,6 +41,7 @@ class OOMODEL_API MetaDefinition : public Super<Declaration>
 {
 	COMPOSITENODE_DECLARE_STANDARD_METHODS(MetaDefinition)
 
+	ATTRIBUTE(Model::TypedList<Declaration>, arguments, setArguments)
```

Comment:
Well `FormalArgument` is specific for methods. What kinds of things do we need to provide here as arguments? Expressions? or do we have different argument types?